pyproject.toml

```
@@ -1,3 +1,92 @@
+# scriv's pyproject.toml
+
+[project]
+name = "scriv"
+description = "Scriv changlog management tool"
+authors = [
+    {name = "Ned Batchelder", email = "ned@nedbatchelder.com"},
+]
+license.text = "Apache-2.0"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+
+requires-python = ">= 3.9"
+
+dynamic = ["readme", "version", "dependencies"]
+
+[project.urls]
+"Mastodon" = "https://hachyderm.io/@nedbat"
+"Funding" = "https://github.com/sponsors/nedbat"
+"Issues" = "https://github.com/nedbat/scriv/issues"
+"Source" = "https://github.com/nedbat/scriv"
+"Home" = "https://github.com/nedbat/scriv"
+"Documentation" = "https://scriv.readthedocs.io"
+
+[project.scripts]
+scriv = "scriv.cli:cli"
+
+[project.optional-dependencies]
+toml = [
+    "tomli; python_version < \"3.11\""
```

You probably want a literal string here:
```suggestion
    'tomli; python_version < "3.11"'
```